packages/wasm/src/address.ts

```diff
@@ -1,14 +1,21 @@
 import { get_short_address_by_index, is_controlled_address } from '../wasm';
-import { AddressIndex } from '@buf/penumbra-zone_penumbra.bufbuild_es/penumbra/core/keys/v1/keys_pb';
+import {
+  Address,
+  AddressIndex,
+} from '@buf/penumbra-zone_penumbra.bufbuild_es/penumbra/core/keys/v1/keys_pb';
 import { JsonValue } from '@bufbuild/protobuf';
+import { bech32Address } from '@penumbra-zone/bech32/src/address';
 
 export const getShortAddressByIndex = (fullViewingKey: string, index: number) =>
   get_short_address_by_index(fullViewingKey, index) as string;
 
-export const isControlledAddress = (
+export const getAddressIndexByAddress = (
```

Comment:
If a consumer is doing `getAddressIndexByAddress()` they are assuming it's controlled right? If so, think we should consider just throwing an error if not instead of returning an undefined.

Reply:
Hm, not necessarily. The `indexByAddress` RPC method uses this helper, and that method can take any address and simply return an empty object if the address isn't controlled.